tests/ember-intl/tests/integration/helpers/format-date-time-range/input-is-falsy-test.ts

```diff
@@ -12,44 +12,45 @@ module(
 
     test('input is 0', async function (assert) {
       await render(hbs`
-      <div data-test-output>
-        {{format-date-time-range 0 0}}
-      </div>
-    `);
+        <div data-test-output>
+          {{format-date-time-range 0 0}}
+        </div>
+      `);
 
       assert
         .dom('[data-test-output]')
-        .hasText(new Intl.DateTimeFormat('en-us').format(0));
+        .hasText(new Intl.DateTimeFormat('en-us').formatRange(0, 0));
```

Comment:
**bugfix**

`format()` wasn't the right method from `Intl.DateTimeFormat` to use. By using `formatRange()`, we also ensure that it works on modern browsers without the need for a polyfill.